lib/rouge/lexers/scala.rb

```diff
@@ -24,8 +25,10 @@ class Scala < RegexLexer
       # negative lookahead to filter out other classes
       op = %r(
         (?!#{whitespace}|#{letter}|#{digits}|#{parens}|#{delims})
-        [\u0020-\u007F\p{Sm}\p{So}]
+        [-!#%&*/:?@\\^\p{Sm}\p{So}]
```

Comment:
man i wish there were a better way to do this in general...

Reply:
agreed